bioio/tests/conftest.py

```diff
@@ -16,19 +16,72 @@
       https://docs.pytest.org/en/latest/plugins.html#requiring-loading-plugins-in-a-test-module-or-conftest-file
 """
 
+import logging
 import pathlib
+import shutil
 import typing
 
+import dask.array as da
+import numpy as np
 import pytest
 
+###############################################################################
+
+log = logging.getLogger(__name__)
+
+###############################################################################
+
 LOCAL_RESOURCES_DIR = pathlib.Path(__file__).parent / "resources"
 LOCAL_RESOURCES_WRITE_DIR = pathlib.Path(__file__).parent / "writer_products"
 
 
-def get_resource_full_path(filename: str) -> typing.Union[str, pathlib.Path]:
-    return LOCAL_RESOURCES_DIR / filename
+def pytest_sessionstart(session: pytest.Session) -> None:
```

Comment:
Actually rethinking this I think I'll change this to use tempdir. Maybe we can write out an `example.txt` to tempdir too and avoid having any files download for this package thus far.